app/Models/Votacao.hs

```diff
@@ -0,0 +1,20 @@
+{-# LANGUAGE OverloadedStrings #-}
+module Models.Votacao where
+import Database.PostgreSQL.Simple
+
+data Votacao = Votacao {
+    id :: Int,
+    dataVotacao:: String,
+    encerrada:: Bool,
+    abstencoes:: Int,
+    nulos:: Int
+ } deriving (Show, Read, Eq)
+
+novaVotacao :: Connection -> String -> Bool -> Int -> Int -> IO()
+novaVotacao conn dataVotacao encerrada abstencoes nulos = do
+    let comando = "INSERT INTO votacao (data,\
+                                       \encerrada,\
+                                       \abstencoes,\
+                                       \nulos) VALUES (?, ?, ?, ?)"
+    execute conn comando (dataVotacao, encerrada, abstencoes, nulos)

```

Comment:
essa linha pode resultar em um erro de sql, que interromperia a execução do programa. uma sugestão pra fazer o tratamento seria usar o `try`:
```hs
    result <- try (execute conn comando (dataVotacao, encerrada, abstencoes, nulos)) :: IO (Either SomeException Int64)
    case result of
        Left err  -> putStrLn $ "Caught exception: " ++ show err
        Right val -> print "Votação cadastrada"
```
o `try` retorna essa entidade do tipo `Either`, que é basicamente pra dizer que o retorno da função dentro do try pode ser tanto uma SomeException (quando resulta em algum erro) quanto um Int64 (que é o que o `execute` retorna se o comando foi executado sem erro).
se for usar esse código, lembra de fazer os imports:
```hs
import Control.Exception
import Data.Int
```